packages/clay-slider/README.mdx

```diff
@@ -6,18 +6,27 @@ packageNpm: '@clayui/slider'
 sibling: 'docs/components/css-slider.html'
 ---
 
-import {Slider} from '$packages/clay-slider/docs/index';
+import {DecadeSlider, Slider} from '$packages/clay-slider/docs/index';
 
 <div class="nav-toc-absolute">
 <div class="nav-toc">
 
+-   [Customization](#customization)
 -   [API](#api)
 
 </div>
 </div>
 
+You can use Slider with just 2 props for it's basic usage, `value` and `onValueChange` which is used to set the value of the Slider.
```

Comment:
```suggestion
Slider is a controlled component and needs just 2 props for its basic use, `value` and `onValueChange`.
```

The props should be self explanatory for what they do, so no need to say "they are used to set the value"